crates/hyperqueue/src/client/commands/journal/mod.rs

```diff
@@ -30,6 +30,9 @@ enum JournalCommand {
 
     /// Connect to a server and remove completed tasks and non-active workers from journal
     Prune,
+
+    /// Connect to a server and forces to flush a journal
+    Flush,
```

Comment:
What's the motivation for this command? It seems weird to have a command that affects the internal implementation details of HQ and essentially flushes a buffer.. I hope it wasn't only added for the Python test xD